.github/workflows/snyk_merge.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: snyk_merge
+
+on:
+  push:
+    branches:
+      - v2
+
+jobs:
+ snyk_clojure:
+   runs-on: ubuntu-latest
+   steps:
+    - uses: twingate/github-action@v1
+      with:
+        service-key: ${{ secrets.TWINGATE_KEY }}
+    - name: checkout the current PR
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        persist-credentials: false
+    - name: Run Clojure Snyk Scan
+      id: scan
+      uses: puppetlabs/security-snyk-clojure-action@v2
+      with:
+        snykToken: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Based on the secrets in this repo, it looks like this should be `SNYK_GENERAL_KEY`?

Reply:
There are a few different syntaxes for this and I'm very confused. Another repo uses env: like this

name: Run Snyk to check for vulnerabilities
        uses: snyk/actions/ruby@master
        env:
          SNYK_TOKEN: ${{ secrets.SNYK_PE_TOKEN }}

Reply:
Also we'll need to add something like this to the scanning step
```
 with:
          command: monitor
          args: --project-name=${{github.repository}}:project.clj
```
Im not sure what Jeremys comment of "right now you can set it using SNYKPROJECT to ${{github.repository}}:pom.xml" was saying exactly

Reply:
`env` sets an environment variable, which I think is redundant in your example, since `secrets.SNYK_PE_TOKEN` is already an environment variable - the secret referenced just needs to correspond with a secret that has been added to the repo. I would guess that later in the workflow you're referencing, they have `snykToken: ${{ secrets.SNYK_TOKEN }}`?

Reply:
Jeremy just made this update to one of our other Clojure repos, so I imagine this should look nearly identical to that: https://github.com/puppetlabs/jruby-deps/pull/33/files

Not sure what that addition is for @KNBAnderson

Reply:
That is supposed to help create something of a foreign key that can be used to link any issues it finds to the repo it's found in. I guess skip for now